plugins/MapLegend.jsx

```diff
@@ -95,9 +97,12 @@ class MapLegend extends React.Component {
         ];
 
         return (
-            <ResizeableWindow extraControls={extraControls} icon="list-alt" initialHeight={this.props.geometry.initialHeight} initialWidth={this.props.geometry.initialWidth}
+            <ResizeableWindow extraControls={extraControls} icon="list-alt"
+                initialHeight={this.props.geometry.initialHeight} initialWidth={this.props.geometry.initialWidth}
                 initialX={this.props.geometry.initialX} initialY={this.props.geometry.initialY}
-                initiallyDocked={this.props.geometry.initiallyDocked} onClose={this.onClose} title={LocaleUtils.trmsg("maplegend.windowtitle")} >
+                initiallyDocked={this.props.geometry.initiallyDocked} dockable={this.props.geometry.side}
```

Comment:
Please keep props sorted or else eslint will complain

Reply:
Ah do you mean, they have to be in alphabetical order?

Reply:
Yes exactly